demo/inferno-router-demo/src/server.ts

```diff
@@ -0,0 +1,164 @@
+import koa from 'koa';; // koa@2
```

Comment:
double semicolon and unnecessary imports

Reply:
WILL FIX

Reply:
Fixed in 8e292ba6